### DISCUSSION TITLE
Plan for Blazor WASM ?

### DISCUSSION THREAD

**Discussion by hkusulja** [microsoft/fluentui#17793] [General] [answered]
Hello,
maybe this is off topic, but i did not find other/better way where to put this question for Microsoft Fluent UI team.
So far, we have Microsoft Fluent UI working based on React which is fine and cool.

However, are there any plans to make all controls, styles of Fluent UI etc, for Blazor native applications with .NET 5 / .NET 6 including server-side and client-side / WASM ?

**Comment by micahgodbolt** [comment, marked as answer]
We have no plans for a Blazor version of Fluent UI at this time.

**Comment by hkusulja** [comment]
@micahgodbolt can you please confirm your statement? based on this documentation:
Use Fluent UI Web Components with Blazor - https://learn.microsoft.com/en-us/fluent-ui/web-components/integrations/blazor

**Comment by micahgodbolt** [comment]
Yes, it looks like you can use web components in Blazor. I'm referring to the notion of building a UI framework specifically for Blazor.
